src/Platform/Microsoft.Testing.Platform/buildMultiTargeting/Microsoft.Testing.Platform.props

```diff
@@ -4,5 +4,12 @@
     <TestingPlatformMetadata Include="ExitOnProcessExitCapability" >
       <Description>--exit-on-process-exit: Close the test process if specific process exits. Process PID must be provided.</Description>
     </TestingPlatformMetadata>
+    <!-- For VS Test Explorer -->
+    <ProjectCapability Include="TestingPlatformServer.ExitOnProcessExitCapability">
```

Comment:
@drognanar these are the capability to check in VS to avoid breaking changes.
In future we will use this way to introduce new features.